project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ object Dependencies {
   // NOTE: remember to change the version numbers in the sample projects
   // when changing them here
 
-  val scalaVersions = Seq("2.13.10") // When updating these also update ci.yml and .mergify.yml
+  val scalaVersions = Seq("2.13.10", "3.1.2") // When updating these also update ci.yml and .mergify.yml
```

Comment:
```suggestion
  val scalaVersions = Seq("2.13.10", "3.3.0-RC3") // When updating these also update ci.yml and .mergify.yml
```

I highly recommend to use 3.3 already. Its final release should happen in April.
Also see [this comment](https://github.com/sbt/sbt/discussions/7194#discussioncomment-5462505) where it's suggested to not use Scala 3.0/3.1/3.2 at this point anymore.
Also 3.3 will be a [Scala LTS release](https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html#our-solution-scala-lts--scala-next) so it might makes sense to stick with that version.